mobile/src/main/java/org/openhab/habdroid/background/ItemsControlsProviderService.kt

```diff
@@ -323,22 +358,23 @@ class ItemsControlsProviderService : ControlsProviderService() {
             else -> return null
         }
 
+        val location = getItemTagLabel(item, allItems, Item.Tag.Location).orEmpty()
+        val equipment = getItemTagLabel(item, allItems, Item.Tag.Equipment).orEmpty()
+
+        val statefulControl = Control.StatefulBuilder(item.name, mainActivityPendingIntent)
+            .setTitle(item.label)
+            .setSubtitle("$location $equipment")
```

Comment:
FWIW, I don't like this, because
- putting location there seems redundant because it's already the zone label
- it may be just me, but at least I already refer to the location in the equipment name.

For me, subtitles will have the form `Wohnzimmer Squeezebox Wohnzimmer` now.

Reply:
IMO both options (include location in equipment name or not) seems valid to me. We should add a setting here to control the content of the subtitle:

* Equipment 
* Location and equipment
* Item name?
* Empty?

I'm not sure if the two options marked with `?` are useful.

Reply:
> I'm not sure if the two options marked with ? are useful.

Empty IMHO isn't, item name seems semi useful to me. What's missing in the list is 'Location only' ... wouldn't that also make sense?

Reply:
I pushed a commit with a settings.

What about making the zone depending on the subtitle? That's for users that don't use a semantic model at all: https://github.com/openhab/openhab-android/pull/2925/commits/ba6ad274f47cebb090efea12f6cf730e5919fd21#diff-45e816015be826032960e39d5cd817fff58308b24645ad9db3af2374d831840fR376-R379

Reply:
I didn't try yet, but sounds like a good idea.